### PR TITLE
Add JD prescription endpoints

### PR DESCRIPTION
**Pull request by palomacione**

### CODE REVIEW

**Comment by palomacione** [comment]
[[Docs] Add prescription upload endpoints to JD](https://trello.com/c/JhiBg0F5/304-docs-add-prescription-upload-endpoints-to-jd)

File: docs/beta_prescriptions_endpoints.md [hunk 1]
```diff
@@ -175,11 +181,11 @@ A json array of prescriptions available in this provider
 [
   {
     "id": "str",
-    "name": "str,"
+    "name": "str"
   },
   {
     "id": "str",
-    "name": "str,"
+    "name": "str"
```
Comment (fabyan-alexander) [inline docs/beta_prescriptions_endpoints.md:188]: Nice :smile:

File: docs/beta_prescriptions_endpoints.md [hunk 2]
```diff
@@ -8,9 +8,12 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 <!-- the following links are referenced throughout this document -->
 [1]: https://github.com/Leaf-Agriculture/Leaf-quickstart-Postman-collection
-[2]: #upload-prescription-raven-slingshot
-[3]: #get-prescription-raven-slingshot
-[4]: #upload-prescription-climate-field-view
+[2]: #upload-prescription-to-raven-slingshot
+[3]: #list-prescriptions-from-raven-slingshot
+[4]: #upload-prescription-to-climate-fieldview
+[5]: #list-prescriptions-from-john-deere
+[6]: #upload-prescription-to-john-deere
```
Comment (fabyan-alexander) [inline docs/beta_prescriptions_endpoints.md:15]: These are hard to keep track

File: docs/beta_prescriptions_endpoints.md [hunk 3]
```diff
@@ -263,4 +269,152 @@ The response is json with the id of the file uploaded to the provider.
...
+]
+}>
+
+  <TabItem value="js">
+
+  ```js
+  const axios = require('axios')
+  const TOKEN = 'YOUR_TOKEN'
+
+  const endpoint ='https://api.withleaf.io/services/beta/prescriptions/api/users/{leafUserId}/johnDeere'
```
Comment (fabyan-alexander) [inline docs/beta_prescriptions_endpoints.md:364]: This endpoint requires the request param **organizationId**

File: docs/beta_prescriptions_endpoints.md [hunk 4]
```diff
@@ -263,4 +269,152 @@ The response is json with the id of the file uploaded to the provider.
...
+
+  </TabItem>
+  <TabItem value="py">
+
+  ```py
+  import requests
+
+  TOKEN = 'YOUR_TOKEN'
+
+  endpoint = 'https://api.withleaf.io/services/beta/prescriptions/api/users/{leafUserId}/johnDeere'
```
Comment (fabyan-alexander) [inline docs/beta_prescriptions_endpoints.md:387]: This endpoint requires the request param **organizationId**

File: docs/beta_prescriptions_endpoints.md [hunk 5]
```diff
@@ -263,4 +269,152 @@ The response is json with the id of the file uploaded to the provider.
...
+  ```
+
+  </TabItem>
+  <TabItem value="sh">
+
+  ```shell
+  curl -X POST \
+      -H 'Authorization: Bearer YOUR_TOKEN' \
+      -F 'file=prescription_rx_map.zip' \
+      'https://api.withleaf.io/services/beta/prescriptions/api/users/{leafUserId}/johnDeere'
```
Comment (fabyan-alexander) [inline docs/beta_prescriptions_endpoints.md:403]: This endpoint requires the request param **organizationId**

File: docs/beta_prescriptions_endpoints.md [hunk 6]
```diff
@@ -263,4 +269,152 @@ The response is json with the id of the file uploaded to the provider.
...
+    "id": "str",
+    "name": "str"
+  }
+]
+```
+### Upload prescription to John Deere
+
+&nbsp<span class="badge badge--success">POST</span> `/users/{leafUserId}/johnDeere`
+
+Upload a prescription using the John Deere credentials of the LeafUserId.
```
Comment (fabyan-alexander) [inline docs/beta_prescriptions_endpoints.md:346]: There is a required request param called **organizationId** for this endpoint to work.
This property should be the id of the organization at JohnDeere

**Review by fabyan-alexander — CHANGES_REQUESTED** [review]
Nice work!
Just a missing property in the upload of prescription to review :smile:

File: docs/beta_prescriptions_endpoints.md (hunk 6 above)
Comment (palomacione) [reply docs/beta_prescriptions_endpoints.md:346]: 04f41a513aba113bba4776f2fb351fd7dc642fe5

**Review by fabyan-alexander — APPROVED** [review]

File: docs/beta_prescriptions_endpoints.md [hunk 7]
```diff
@@ -263,4 +269,154 @@ The response is json with the id of the file uploaded to the provider.
...
+  print(response.json())
+  ```
+
+  </TabItem>
+  <TabItem value="sh">
+
+  ```shell
+  curl -X GET \
+      -H 'Authorization: Bearer YOUR_TOKEN' \
+      -d '{ "name": "str"}' \
```
Comment (ricardosantacattarina) [inline docs/beta_prescriptions_endpoints.md:319]:
```suggestion
```
I remember we discussed about what should be in the body of a GET request, I think we should remove every case of this -d parameter on GET requests.

**Review by ricardosantacattarina — APPROVED** [review]

**Review by Vinniekun — APPROVED** [review]

File: docs/beta_prescriptions_endpoints.md (hunk 7 above)
Comment (fabyan-alexander) [reply docs/beta_prescriptions_endpoints.md:319]: +1

Comment (palomacione) [reply docs/beta_prescriptions_endpoints.md]: 280dd7a66d40a689cfe43af47d947c3fbc86ed46

**Review by fabyan-alexander — APPROVED** [review]

**Pull request merged**